Moderation-Policy.md

```diff
@@ -31,10 +31,11 @@ Any alternative Moderation Policy used for a given repository must be included i
 
 ### Terms
 
-* *Collaborator* refers to any individual with configured access permissions to any Node.js GitHub organization repository. See [GitHub's access permissions documentation](https://help.github.com/articles/what-are-the-different-access-permissions/) for more information.
+* *Collaborator* refers to any individual with configured write (commit) permissions to any Node.js GitHub organization repository *other than the Moderation Repository*. See [GitHub's access permissions documentation](https://help.github.com/articles/what-are-the-different-access-permissions/) for more information.
```

Comment:
I could be wrong, but I thought that being a member of the org gave read access, but not write.

On further reflection though, if this is the case, it doesn't make the wording here wrong. Absolute worst case it's a bit superfluous, and best case it makes an important distinction, so there's no downside. I'm ok with this.